app/src/app/admin/articles/article/meta/meta.spec.ts

```diff
@@ -31,6 +17,29 @@ namespace app.admin.articles.article.meta {
                 query: sinon.stub()
             };
 
+        article._metas = [
+            new common.models.Meta({
+                metaName: 'name',
+                metaContent: 'foo',
+                metaableId: article.articleId
+            }),
+            new common.models.Meta({
+                metaName: 'description',
+                metaContent: 'bar',
+                metaableId: article.articleId
+            }),
+            new common.models.Meta({
+                metaName: 'keyword',
+                metaContent: 'foo, bar',
+                metaableId: article.articleId
+            }),
+            new common.models.Meta({
+                metaName: 'canonical',
+                metaContent: 'https://foo.bar.com',
+                metaableId: article.articleId
+            })
+        ];
```

Comment:
Why move it out of the mock? It will mean that the new _metas will be marked as not existing, when the mock might have been marked as existing.